### PR TITLE
docs: add information on how to fork gitops-template properly

### PR DESCRIPTION
**Pull request by fharper**

### CODE REVIEW

File: docs/explore/gitops.md [hunk 1]
```diff
@@ -56,3 +56,17 @@ The metaphor example application includes an example of how easy it is to set di
...
 Any Helm value that should deviate from the default chart can be set here so your environment overrides are always concise and clear.
+
+## Using Your Own gitops-template Repository Fork
+
+If you want to customize the template before creating a new cluster, you can fork the [gitops-template repository](https://github.com/kubefirst/gitops-template), and use it as the source for cluster creation. To do so, follow the [forking instructions from GitHub](https://docs.github.com/en/get-started/quickstart/fork-a-repo). Note that the tags won't follow, and they are needed for the CLI to properly function. [Clone your new repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) locally, and once inside the repository folder, run the following commands in your terminal to update your repository with the tags:
+
+```shell
+# Replace <yourusername> by your GitHub username or the organization into which you fork gitops-template
+# If you give another name to your gitops-template repository, update it also
+git remote add upstream git@github.com:yourusername/gitops-template.git
```
Comment (jarededwards) [inline docs/explore/gitops.md]:
```suggestion
git remote add upstream git@github.com:<yourusername>/gitops-template.git
```

File: versioned_docs/version-2.0/explore/gitops.md [hunk 2]
```diff
@@ -56,3 +56,17 @@ The metaphor example application includes an example of how easy it is to set di
...
 Any Helm value that should deviate from the default chart can be set here so your environment overrides are always concise and clear.
+
+## Using Your Own gitops-template Repository Fork
+
+If you want to customize the template before creating a new cluster, you can fork the [gitops-template repository](https://github.com/kubefirst/gitops-template), and use it as the source for cluster creation. To do so, follow the [forking instructions from GitHub](https://docs.github.com/en/get-started/quickstart/fork-a-repo). Note that the tags won't follow, and they are needed for the CLI to properly function. [Clone your new repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) locally, and once inside the repository folder, run the following commands in your terminal to update your repository with the tags:
+
+```shell
+# Replace <yourusername> by your GitHub username or the organization into which you fork gitops-template
+# If you give another name to your gitops-template repository, update it also
+git remote add upstream git@github.com:yourusername/gitops-template.git
```
Comment (jarededwards) [inline versioned_docs/version-2.0/explore/gitops.md]:
```suggestion
git remote add upstream git@github.com:<yourusername>/gitops-template.git
```

**Comment by fharper** [comment]
@jarededwards will add more information about this.

**Comment by fharper** [comment]
@johndietz will be the one updating this PR today.

**Review by johndietz — DISMISSED** [review]

**Review by jarededwards — APPROVED** [review]

**Pull request merged**